bitnami/thanos/values.yaml

```diff
@@ -2762,6 +2762,9 @@ ruler:
   ##
   dnsDiscovery:
     enabled: true
+  ## @param ruler.query_url Thanos query/query-frontend URL to link in Ruler UI.
+  ##
+  query_url: http://thanos-query:9090
```

Comment:
I think is not needed. Also it could infer to error if we deploy the chart with a name different than `thanos`

Reply:
I thought we need to supply a default value, then should I keep it empty?

Reply:
We can keep it empty as I mentioned before. Please follow camel case (`queryURL` instead of `query_url`) for this name according to the name convention in this file

Reply:
Sure!